### PR TITLE
code review

### PR DESCRIPTION
**Pull request by xash3000**

### CODE REVIEW

File: conduit/app.py [hunk 1]
```diff
@@ -9,7 +9,8 @@
 
 
 def create_app(config_object=ProdConfig):
-    """An application factory, as explained here: http://flask.pocoo.org/docs/patterns/appfactories/.
```
Comment (xash3000) [inline conduit/app.py:12]: long line (> 79)

File: conduit/app.py [hunk 2]
```diff
@@ -30,22 +31,22 @@ def register_extensions(app):
     db.init_app(app)
     migrate.init_app(app, db)
     jwt.init_app(app)
-    return None
```
Comment (xash3000) [inline conduit/app.py:33]: no need to return None

File: conduit/articles/views.py [hunk 3]
```diff
@@ -48,7 +48,7 @@ def make_article(body, title, description, tagList=None):
     try:
         article = Article(title=title, description=description, body=body,
                           author=current_identity.profile)
-    except IntegrityError as e:
```
Comment (xash3000) [inline conduit/articles/views.py:51]: unused variable e

File: conduit/commands.py [hunk 4]
```diff
@@ -53,7 +53,7 @@ def clean():
 
     Borrowed from Flask-Script, converted to use Click.
     """
-    for dirpath, dirnames, filenames in os.walk('.'):
```
Comment (xash3000) [inline conduit/commands.py:56]: unused variable dirnames

File: conduit/exceptions.py [hunk 5]
```diff
@@ -12,17 +12,16 @@ def __init__(self, message, status_code=None, payload=None):
         self.payload = payload
 
     def to_json(self):
-        rv = dict(self.payload or ())
```
Comment (xash3000) [inline conduit/exceptions.py:15]: variable override

File: conduit/exceptions.py [hunk 6]
```diff
@@ -12,17 +12,16 @@ def __init__(self, message, status_code=None, payload=None):
...
         rv = self.message
         return jsonify(rv)
 
 
-def TEMPLATE(data, code=500):
```
Comment (xash3000) [inline conduit/exceptions.py:20]: function names should be lowercase

File: tests/conftest.py [hunk 7]
```diff
@@ -52,10 +52,10 @@ def db(app):
 @pytest.fixture
 def user(db):
     """A user for the tests."""
-    class user():
```
Comment (xash3000) [inline tests/conftest.py:55]: class names should be uppercase

File: tests/test_config.py [hunk 8]
```diff
@@ -8,11 +8,11 @@ def test_production_config():
     """Production config."""
     app = create_app(ProdConfig)
     assert app.config['ENV'] == 'prod'
-    assert app.config['DEBUG'] is False
```
Comment (xash3000) [inline tests/test_config.py:11]: don't use is with Boolean values

File: tests/test_models.py [hunk 9]
```diff
@@ -162,8 +162,8 @@ def test_make_comment(self, user):
         comment = Comment(article, user.profile, 'some body')
         comment.save()
 
-        assert comment.article is article
-        assert comment.author is user.profile
+        assert comment.article == article
```
Comment (xash3000) [inline tests/test_models.py:165]: don't use is when comparing two values

**Comment by realazizk** [comment]
Thank you very much.

**Pull request merged**